packages/animation/src/utils/getGlobalSpace.js

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * @typedef {import('../../edit-story/types').Element} Element
+ * @typedef {import('../../../../assets/src/edit-story/types').Element} Element
```

Comment:
@swissspidy Not sure what to do about this one.

Reply:
As mentioned on the ticket, ideally we'd import these types from a dedicate package.

For the time being, we could just _copy_ the whole type def from `edit/story/types.js` to this file. Adding a `TODO` comment indicating that this should be improved in the future.